phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/scriptui/StepRepresentationTextBox.java

```diff
@@ -42,7 +43,7 @@ protected void drawStepContent() {
 
         textField = new TextField();
         textField.textProperty().addListener((observable, oldValue, newValue) -> {
-            if (newValue.trim().equals("")) {
+            if (StringUtils.isBlank(newValue.trim())) {
```

Comment:
As far as I understand `StringUtils.isBlank` a trim isn't needed anymore. 
Can you please take a look if this is the case? If this is the case please remove the `trim()` call.